plugins/wallabag.koplugin/main.lua

```diff
@@ -277,6 +282,14 @@ function Wallabag:addToMainMenu(menu_items)
                                     self.is_delete_read = not self.is_delete_read
                                     self:saveSettings()
                                 end,
+                            },
+                            {
+                                text = _("Remotely delete on hold articles"),
```

Comment:
Wouldn't "articles on hold" sound better? (Same for the other texts.)

Reply:
Yes, it sounds better. Changed.